src/components/MessagesList/MessagesList.vue

```diff
@@ -358,6 +358,7 @@ export default {
 
 		areGroupsIdentical(group1, group2) {
 			if (group1.messages.length !== group2.messages.length
+				|| JSON.stringify(group1.messages) !== JSON.stringify(group2.messages)
```

Comment:
So if there are 3 messages in a group and the second message was edited, it should split the group like
group 1 : message 1
group 2 : message 2 and message 3 ?

or this would split into 3 groups ? 

I think computation cost would still be considered reasonable as there are usually 1 to 10 messages per group but if the idea was to split edited message to new group, then we can skip this comparison.

Reply:
Should be two groups, as in your example :+1:

Reply:
I think having 3 groups would also be okay, so it's clear that the 3rd messages was not edited by the other person.

And then basically the rule would be `if message is edited, never group`